### ISSUE TITLE
Release 1.20.1 from hex missed VERSION file referenced in mix.exs

### ISSUE THREAD

**Issue by samuelventura** [elixir-mint/castore#50]
```bash
Error while loading project :castore at /home/samuel/dpi_ws/uriel_counter/deps/castore
** (File.Error) could not read file "VERSION": no such file or directory
```

mix.lock
```elixir
  "castore": {:hex, :castore, "0.1.21", "f618df4d83ad844979e2748cb76937d970ea2c1aee3ca96d4c8301884b9939a1", [:mix], [], "hexpm", "1e4c72e5c476e50deee88ec1e6949f804c9a410d813dc17d3ce78b9b7c55b289"},
```

mix.exs
```elixir
defmodule CAStore.MixProject do
  use Mix.Project

  @repo_url "https://github.com/elixir-mint/castore"

  def project do
    [
      app: :castore,
      version: version(),
      elixir: "~> 1.0",
      start_permanent: Mix.env() == :prod,
      deps: deps(),
      xref: [exclude: [:public_key]],

      # Hex
      package: package(),
      description: "Up-to-date CA certificate store.",

      # Docs
      name: "CAStore",
      docs: [
        source_ref: "v#{version()}",
        source_url: @repo_url
      ]
    ]
  end

  def application do
    [
      extra_applications: [:logger]
    ]
  end

  defp deps do
    [
      {:ex_doc, "~> 0.22", only: :dev}
    ]
  end

  defp package do
    [
      files: ["lib/castore.ex", "priv", "mix.exs", "README.md"],
      licenses: ["Apache-2.0"],
      links: %{"GitHub" => @repo_url}
    ]
  end

  defp version do
    "VERSION"
    |> File.read!()
    |> String.trim()
  end
end

```

**Comment by samuelventura** [comment]
Hex shows different checksum.

- https://hex.pm/packages/castore/0.1.21
- 1e4c72e5c476e50deee88ec1e6949f804c9a410d813dc17d3ce78b9b7c55b289

**Comment by Nyx1197** [comment]
Same problam, create "VERSION" file in /deps/castore/, and compile ok.

**Comment by NickTaylor-** [comment]
Yeah, I just did `echo "1.20.1" > deps/catstore/VERSION` as a stopgap.

**Comment by Nyx1197** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by Nyx1197** [comment]
i already send a pull request and wait respon

**Comment by ntmonson** [comment]
Until the PR is merged and a new version is pushed to hex you can _probably_ use 

`{:castore, "== 0.1.20", override: true}` 

in your deps to keep packages from updating castore to 0.1.21.  This will work in your ci/cd pipelines where manually adding the file to the deps folder will not.

**Comment by Nyx1197** [comment]
> 

thanks, that's help a lot

**Comment by ArthurClemens** [comment]
> Yeah, I just did `echo "0.1.21" > deps/catstore/VERSION` as a stopgap.

That should be `echo "0.1.21" > deps/castore/VERSION`

**Comment by adamu** [comment]
> Hex shows different checksum.
> 
>   * https://hex.pm/packages/castore/0.1.21
>   * 1e4c72e5c476e50deee88ec1e6949f804c9a410d813dc17d3ce78b9b7c55b289

Indeed, the [GitHub tagged version](https://github.com/elixir-mint/castore/blob/v0.1.21/mix.exs) does not require `VERSION`, so it appears a different file was included in the hex package.

**Comment by ericmj** [comment]
We have released 0.1.22 that matches what's on github.
